test/global-override.xspec

```diff
@@ -1,47 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<x:description stylesheet="global.xsl" xmlns:mirror="x-urn:test:mirror"
-	xmlns:x="http://www.jenitennison.com/xslt/xspec">
-
-	<x:helper stylesheet="mirror.xsl" />
+<x:description stylesheet="global-override.xsl" xmlns:x="http://www.jenitennison.com/xslt/xspec">
 
 	<x:param name="global-param" select="'global param overridden by XSpec'" />
 
 	<x:variable name="global-variable" select="'global variable overridden by XSpec'" />
 
-	<x:scenario label="With x:context">
-		<x:context select="//context-grandchild">
-			<context-child>
-				<context-grandchild />
-			</context-child>
-		</x:context>
-
-		<x:scenario label="global x:param in XSpec">
-			<x:call template="get-global-param" />
-			<x:expect label="overrides xsl:param in SUT" select="'global param overridden by XSpec'"
-			 />
-		</x:scenario>
-
-		<x:scenario label="global x:variable in XSpec">
-			<x:call template="get-global-variable" />
-			<x:expect label="overrides xsl:variable in SUT"
-				select="'global variable overridden by XSpec'" />
-		</x:scenario>
-
-		<x:scenario label="template context">
-			<x:scenario label="apply-templates invocation">
-				<x:context mode="mirror:context-mirror" />
-				<x:expect label="x:context/@select">
-					<context-grandchild />
-				</x:expect>
-			</x:scenario>
+	<x:scenario label="Global x:param in XSpec">
+		<x:call template="get-global-param" />
+		<x:expect label="overrides xsl:param in SUT" select="'global param overridden by XSpec'" />
+	</x:scenario>
 
-			<x:scenario label="call-template invocation">
-				<x:call template="mirror:context-mirror" />
-				<x:expect label="x:context/@select">
-					<context-grandchild />
-				</x:expect>
-			</x:scenario>
-		</x:scenario>
+	<x:scenario label="Global x:variable in XSpec">
+		<x:call template="get-global-variable" />
+		<x:expect label="overrides xsl:variable in SUT"
```

Comment:
In a separate PR, do you think it's worthwhile to test that a global `xsl:variable` in the SUT can be overridden by a global `x:param` in the test file? The wiki https://github.com/xspec/xspec/wiki/Writing-Scenarios#global-parameters documents `x:param` as overriding globals in the SUT, whether it's a global parameter or global variable.

Also, this file and `template-context.xspec` aren't related to `run-as=external`, are they? They look like independent refactoring for clarity.

Reply:
> In a separate PR, do you think it's worthwhile to test that a global `xsl:variable` in the SUT can be overridden by a global `x:param` in the test file?

Done in 550804f35f9d8d8c6227de2aa7d753efedf0e4bc.
(`external_global-override.xspec` and `global-override.xspec` have been consolidated and moved to `test/end-to-end/cases/`. So, done in this PR.)

> Also, this file and `template-context.xspec` aren't related to `run-as=external`, are they? They look like independent refactoring for clarity.

[`external_template-context.xspec`](https://github.com/xspec/xspec/blob/e2de623afe020f777b7f7ca04301f829d10ee899/test/external_template-context.xspec) imports `template-context.xspec`.
[`external_global-override.xspec`](https://github.com/xspec/xspec/blob/39b87c1123bd264711d3da7f1f1f1bdb9055bc2c/test/external_global-override.xspec) was a slightly modified copy of [`global-override.xspec`](https://github.com/xspec/xspec/blob/39b87c1123bd264711d3da7f1f1f1bdb9055bc2c/test/global-override.xspec) (although 550804f35f9d8d8c6227de2aa7d753efedf0e4bc has consolidated them).
That's why this PR contains `template-context.xspec` and `global-override.xspec` along with `run-as=external` files.

Reply:
> That's why this PR contains

Thanks. I hadn't looked at the whole set of files and didn't make the connection. This PR looks good to me.